js/src/components/paid-ads/asset-group/asset-group-card.js

```diff
@@ -138,14 +138,15 @@ export default function AssetGroupCard() {
 						subheading={
 							<>
 								{ spec.subheading }
-								{ index === 0 && isSelectedFinalUrl && (
-									<ExternalLink href="https://support.google.com/google-ads/answer/6167101">
-										{ __(
-											'Learn how to write effective ads',
-											'google-listings-and-ads'
-										) }
-									</ExternalLink>
-								) }
+								{ spec.key === ASSET_FORM_KEY.HEADLINE &&
+									isSelectedFinalUrl && (
+										<ExternalLink href="https://support.google.com/google-ads/answer/6167101">
+											{ __(
+												'Learn how to write effective ads',
+												'google-listings-and-ads'
+											) }
+										</ExternalLink>
+									) }
 							</>
```

Comment:
💅  We could potentially add this in the specs as props. (I mean the link and the link label) 

```jsx
 { spec.linkUrl && spec.linkLabel && isSelectedFinalUrl && (
        <ExternalLink href={linkUrl}>
...
```

Reply:
Good idea. I moved all JSX of the link to the `assetSpecs` module by https://github.com/woocommerce/google-listings-and-ads/pull/1908/commits/465f11ea01c8d09dba5618ba12a404c58227ea54.